src/fr/composants/code.md

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
And same comment here, looks like this was supposed to be part of the checkbox folder.